src/main/java/com/example/EnumDemo/enumC/Type.java

```diff
@@ -0,0 +1,22 @@
+package com.example.EnumDemo.enumC;
+
+public enum Type {
+    TSHIRT("Tshirt"),
+    FORMAL("Formal"),
+    JACKET("Jacket"),
+    JERKIN("Jerkin"),
+    POLO("Polo"),
+    HOODY("Hoody"),
+    TURTLENECK("Turtleneck");
```

Comment:
```suggestion
    TURTLE_NECK("Turtleneck");
```
Please check the enum naming convention

Reply:
Done